src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -183,13 +197,50 @@ private AnalyzeEntitiesResponse analyzeEntitiesText() throws IOException {
     return entities;
   }
 
+  public AnalyzeSyntaxResponse analyseSyntaxText() throws IOException {
+    try (LanguageServiceClient language = LanguageServiceClient.create(getSettings())) {
+      Document doc = Document.newBuilder().setContent(message).setType(Type.PLAIN_TEXT).build();
+      AnalyzeSyntaxRequest request =
+           AnalyzeSyntaxRequest.newBuilder()
+                .setDocument(doc)
+                .setEncodingType(EncodingType.UTF16)
+                .build();
+      
+      return language.analyzeSyntax(request);
+    }
+  }
+
+  public Set<String> getAdjectives() throws IOException {
+    Set<String> adjectives = new LinkedHashSet<String>();
+
+    for (Token token : analyseSyntaxText().getTokensList()) {
+      String partOfSpeech = token.getPartOfSpeech().getTag().toString();
+      if(partOfSpeech.equals("ADJ")) {
+        adjectives.add(token.getLemma().toLowerCase());
+      }
+    }
+
+    return adjectives;
+  }
+
+  public String checkInjection() {
+    if (message.indexOf("<script>") != -1 || message.indexOf("</script>") != -1 ||
+        message.indexOf("<html>") != -1 || message.indexOf("</html>") != -1) {
+      return "html-injection";
+    }
+
+    return "no-html-injection";
```

Comment:
please use constant or enum or exception, not plain string in this case.

Reply:
Mara's. She can fix the changes after merging. But I guess her code was already merged, I just needed it to not have merging conflicts